guides/common/modules/proc_recovering-from-disaster-with-two-active-project-servers.adoc

```diff
@@ -0,0 +1,23 @@
+[id="recovering-from-disaster-with-two-active-project-servers"]
+= Recovering from disaster with two active {ProjectServer}s
+
+If the health checks implemented in xref:preparing-for-disaster-recovery-with-two-active-project-servers[] report an issue on one of your {ProjectServer}s, it might mean that the server has failed.
+If the server is down, you must re-register hosts to the other server.
+
+.Procedure
+. Verify the status of the server:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# {foreman-maintain} health check
+----
+. If `{foreman-maintain} health check` reported a problem, ensure that the server is powered off.
+// Do we need a step to disable ISS if that's what users are using to ensure consistency in content?
```

Comment:
A question here, as a comment on line 15.

Reply:
I do not think so.

Reply:
Thanks, removing the comment.